### PR TITLE
Update the page on "Amazon DynamoDB Connector Example"

### PR DESCRIPTION
**Pull request by poorna2152**
## Purpose
$subject

### CODE REVIEW

File: en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 1]
```diff
@@ -36,21 +36,15 @@ The following diagram shows the overall solution. The user creates a table, stor
 
 If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
-## Configure the connector in WSO2 Integration Studio
+## Setting up the connector in WSO2 Integration Studio
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:39]: Why keeping integration studio content?

File: en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 2]
```diff
@@ -36,21 +36,15 @@ The following diagram shows the overall solution. The user creates a table, stor
...
-Follow these steps to set up the Integration Project and the Connector Exporter Project.  
-
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!} 
-
 ### Add integration logic
 
-First create an API, which will be where we configure the integration logic. Right click on the created Integration Project and select, **New** -> **Rest API** to create the REST API. Specify the API name as `amazonDynamoDBAPI` and API context as `/resources`.
-    
-<img src="{{base_path}}/assets/img/integrate/connectors/adding-an-api.jpg" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
+First create an API, which will be where we configure the integration logic. Select Micro Integrator and click on `+` in APIs to create a REST API. Specify the API name as `amazonDynamoDBAPI` and API context as `/resources`.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:45]: This sentence is not that meaningful `First create an API, which will be where we configure the integration logic.`

Comment (anupama-pathirage) [inline en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:45]:
```suggestion
First create an API, which will be where we configure the integration logic. Select Micro Integrator and click on `+` in APIs to create a REST API. Specify the API name as `amazonDynamoDBAPI` and the API context as `/resources`.
```

File: en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 3]
```diff
@@ -60,79 +54,79 @@ Now follow the steps below to add resources to the API.
...
 1. Initialize the connector.
     
-    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Amazondynamodb Connector** section. Then drag and drop the `init` operation into the Design pane.
-        
-        <img src="{{base_path}}/assets/img/integrate/connectors/amazon-dynamodb-api-init.jpg" title="Drag and drop init operation" width="500" alt="Drag and drop init operation"/>   
+    1. Navigate into the **Connectors** pane and select the `init` operation under the **Amazondynamodb Connector**.
+
+        <img src="{{base_path}}/assets/img/integrate/connectors/amazon-dynamodb/amazon-dynamodb-api-init.png" title="Drag and drop init operation" width="800" alt="Drag and drop init operation"/>   
     
     2. Add the property values into the `init` operation as shown below. Replace the `region`, `accessKeyId`, `secretAccessKey`, `blocking` with your values.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:61]:
```suggestion
    2. Add the property values into the `init` operation as shown below. Replace the `region`, `accessKeyId`, `secretAccessKey`,  and `blocking` with your values.
```

File: en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 4]
```diff
@@ -60,79 +54,79 @@ Now follow the steps below to add resources to the API.
...
+
+        <img src="{{base_path}}/assets/img/integrate/connectors/amazon-dynamodb/amazon-dynamodb-drag-and-drop-create-table-parameters.png" title="Drag and drop create table operation" width="800" alt="Drag and drop create table operation"/> 
+
+    3. To get the input values in to the API we can use the [property mediator]({{base_path}}/reference/mediators/property-mediator). Navigate into the **Mediators** pane and select the `Property` mediator.
+
+        <img src="{{base_path}}/assets/img/integrate/connectors/amazon-dynamodb/amazon-dynamodb-drag-and-drop-property-mediator.png" title="Add property mediators" width="800" alt="Add property mediators"/>
 
         The parameters available for configuring the Property mediator are as follows:
     
         > **Note**: That the properties should be add to the pallet before create the operation.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:94]: `That the properties should be add to the pallet before create the operation.` This sentence is not complete and grammatically correct. 
It should be
`**Note**: The properties should be added to the pallet before creating the operation.`

File: en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 5]
```diff
@@ -142,174 +136,164 @@ Now follow the steps below to add resources to the API.
...
+        <img src="{{base_path}}/assets/img/integrate/connectors/amazon-dynamodb/amazon-dynamodb-drag-and-drop-put-items.png" title="Drag and drop create put items operation" width="800" alt="Drag and drop put items operation"/>    
 
     2. The putItem operation use to insert new items to the tables. `putItem` operation parameters listed here.
                
-       - **item** : A map of attribute name/value pairs, one for each attribute. Only the primary key attributes are required, but you can optionally provide other attribute name-value pairs for the item 
-       - **tableName** :  The name of the table to contain the item. 
+        - **item** : A map of attribute name/value pairs, one for each attribute. Only the primary key attributes are required, but you can optionally provide other attribute name-value pairs for the item 
+        - **tableName** :  The name of the table to contain the item. 
         
         While invoking the API, the above two parameter values come as a user inputs.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:148]:
```suggestion
        While invoking the API, the above two parameter values come as a user input.
```

File: en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 6]
```diff
@@ -9,22 +9,22 @@ Given below is a sample scenario that demonstrates how to work with the Amazon D
...
 5. Remove inserted employee details (items).
-6. Retrieve list of tables.
-7. Remove created employee details table.
+6. Retrieve the list of tables.
+7. Remove the created employee details table.
 
-All seven operations are exposed via an API. The API with the context `/resources` has seven resources
+All seven operations are exposed via an API. The API with the context `/resources` has seven resources.
 
 * `/addtable` : Creates a new table in the Amazon DynamoDB with the specified table name to store employee details.
```
Comment (chanikag) [inline en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:21]:
```suggestion
* `/addtable` : Create a new table in the Amazon DynamoDB with the specified table name to store employee details.
```

File: en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 7]
```diff
@@ -9,22 +9,22 @@ Given below is a sample scenario that demonstrates how to work with the Amazon D
...
 * `/insertdetails` : Insert employee data (items) and store in the specified table.
 * `/updatetable` : Update specified table (provisioned throughput settings, global secondary indexes, or DynamoDB Streams settings for a specified table).
```
Comment (chanikag) [inline en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:23]:
```suggestion
* `/updatetable` : Update the specified table (provisioned throughput settings, global secondary indexes, or DynamoDB Streams settings for a specified table).
```

File: en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 8]
```diff
@@ -142,174 +136,164 @@ Now follow the steps below to add resources to the API.
...
     
-    3. Then drag and drop the `Property` mediators into the Design pane as mentioned in `addtable` operation. The parameters available for configuring the Property mediator are as follows.
-        
-       Add the property mediator to capture the `exclusiveStartTableName` value.  Please follow the steps given in the `listTables` operation.
-   
-       - **name** : exclusiveStartTableName
-       - **expression** : json-eval($.exclusiveStartTableName)
+    3. Select the `Property` mediator as mentioned in `addtable` operation. Add the property mediator to capture the `exclusiveStartTableName` value.
+
+        - **name** : exclusiveStartTableName
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:286]: Use backticks for values on rhs

**Review by chanikag — APPROVED** [review]

File: en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 9]
```diff
@@ -518,26 +479,15 @@ You can download the ZIP file and extract the contents to get the project code.
```
Comment (chanikag) [inline en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:479]: Please add the modified zip file

**Pull request merged**